### PR TITLE
feat: add proxy command for sandbox port forwarding with review fixes

### PR DESCRIPTION
**Pull request by keithfx999**
## Summary

Adds `ags proxy`, a new subcommand that forwards a remote sandbox port to localhost — similar to `kubectl port-forward`. Both HTTP and WebSocket protocols are fully supported, making it suitable for web dev servers (Vite, webpack-dev-server), API servers, and any HTTP-based service running inside a sandbox.

```bash
# Forward sandbox port 8080 to localhost:8080
ags proxy sandbox-xxx 8080

# Forward sandbox port 8080 to a different local port
ags proxy sandbox-xxx 3000:8080

# Bind to all interfaces (with a security warning)
ags proxy sandbox-xxx 8080 --address 0.0.0.0
```

## How it works

- Starts a local HTTP server that acts as a reverse proxy to `https://<port>-<sandbox-id>.<region-domain>`
- Automatically injects the sandbox access token (`X-Access-Token`) into every proxied request
- WebSocket upgrade requests are detected per RFC 6455 and tunnelled via a bidirectional gorilla/websocket bridge
- The access token is acquired once at startup; its lifetime is bound to the sandbox instance lifecycle so no refresh or 401-retry logic is needed

## Changes

| File | Change |
|------|--------|
| `cmd/proxy.go` | CLI entry point: argument parsing, address validation, signal handling |
| `cmd/proxy_test.go` | Unit tests for `parsePortSpec` (18 cases incl. edge cases) |
| `internal/proxy/proxy.go` | Core reverse proxy: HTTP forwarding, WebSocket bridge, graceful shutdown |
| `internal/proxy/proxy_test.go` | Integration tests: HTTP/WS forwarding, concurrency, Stop-with-active-WS, verbose logging |
| `internal/repl/repl.go` | REPL autocomplete and help text |
| `docs/ags-proxy{,-zh}.md` | New bilingual reference page |
| `README{,-zh}.md`, `docs/ags{,-zh}.md` | Feature list and command table updated |

## Security

- Binds to `127.0.0.1` by default; prints an explicit warning when `--address` is set to a non-loopback address
- In non-verbose mode, proxy errors only return `"Bad Gateway"` to the client — internal host names and TLS details are never exposed
- Token is injected via header only, never in URLs or logs

## Known Limitations

- WebSocket Ping/Pong control frames are handled internally by the proxy (gorilla/websocket behaviour) and are not forwarded end-to-end
- The access token is fetched once at startup; if the sandbox is destroyed and recreated while the proxy is running, restart `ags proxy` to pick up the new token

## Test Plan

- [x] `go test ./internal/proxy/... ./cmd/...` — all pass
- [x] `golangci-lint run ./...` — 0 issues
- [x] `gofmt -d .` — no diff
- [x] Manual: `ags proxy <sandbox-id> 8080` against a live sandbox, verified HTTP and WebSocket traffic forwarded correctly

### CODE REVIEW

File: cmd/proxy.go [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+into all proxied requests.
+
+Both HTTP and WebSocket protocols are fully supported, making this suitable for
+web development servers (e.g., Vite, webpack-dev-server), API servers, and any
+HTTP-based service running in the sandbox.
+
+NOTE: The remote port must be configured as accessible in the sandbox console.
+
+Port Syntax:
+  <remote_port>                Forward remote port to the same local port
```
Comment (kofj) [inline cmd/proxy.go:44]: 没看明白这个是什么意思？

Comment (keithfx999) [reply cmd/proxy.go:44]: 这个是当前沙箱只能访问特定的几个端口，所以这里有提示
